Corrected hyperlink address in help manual -h, -c option by removing the unnecessary bracket.

creaturenex opened the pull request:
<!--
About this template

The following template aims to help contributors write a good description for their pull requests.
We'd like you to provide a description of the changes in your pull request (i.e. bugs fixed or features added), motivation behind the changes, and complete the checklist below before opening a pull request.

Feel free to discard it if you need to (e.g. when you just fix a typo). -->

### Motivation / Background

<!-- Describe why this Pull Request needs to be merged. What bug have you fixed? What feature have you added? Why is it important? -->

This Pull Request has been created because of issue #45988

### Detail

This Pull Request corrects the link address to access the css-bundling readme.

### Additional information

Simple fix, needs to be tested. 

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] There are no typos in commit messages and comments.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Feature branch is up-to-date with `main` (if not - rebase it).
* [x] Pull request only contains one commit for bug fixes and small features. If it's a larger feature, multiple commits are permitted but must be descriptive.
* [ ] Tests are added if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.
* [ ] PR is not in a draft state.
* [ ] CI is passing.

<!--
Note: Please avoid making *Draft* pull requests, as they still send
notifications to everyone watching the Rails repo.
Create a pull request when it is ready for review and feedback
from the Rails team :).

If your pull request affects documentation or any non-code
changes, guidelines for those changes are [available
here](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#contributing-to-the-rails-documentation)

Thanks for contributing to Rails! -->

creaturenex committed:
Corrected hyperlink address by removing the unneccesary bracket

rafaelfranca commented:
Thank you for the pull request but the code is correct. The bracket is open to list the options and closed after the link. I think this is mostly a bug in the hyperlink parser in your terminal that is recoginizing the bracket as part of the link. On VSCode terminal, iterm and Terminal.app this link can be open without error.

rafaelfranca commented on railties/lib/rails/generators/rails/app/app_generator.rb:
```diff
@@ -264,7 +264,7 @@ class AppGenerator < AppBase
       class_option :minimal, type: :boolean, desc: "Preconfigure a minimal rails app"
       class_option :javascript, type: :string, aliases: ["-j", "--js"], default: "importmap", desc: "Choose JavaScript approach [options: importmap (default), webpack, esbuild, rollup]"
-      class_option :css, type: :string, aliases: "-c", desc: "Choose CSS processor [options: tailwind, bootstrap, bulma, postcss, sass... check https://github.com/rails/cssbundling-rails]"
+      class_option :css, type: :string, aliases: "-c", desc: "Choose CSS processor [options: tailwind, bootstrap, bulma, postcss, sass... check https://github.com/rails/cssbundling-rails"
```
```suggestion
      class_option :css, type: :string, aliases: "-c", desc: "Choose CSS processor [options: tailwind, bootstrap, bulma, postcss, sass] check https://github.com/rails/cssbundling-rails for more options"
```

rafaelfranca commented:
Let's fix this in another way.

creaturenex committed:
Update railties/lib/rails/generators/rails/app/app_generator.rb

This correction should work regardless of which hyperlink parser or terminal being used.

Co-authored-by: Rafael Mendonça França <rafael@franca.dev>

creaturenex commented:
Awesome, I just tested in VSCode and it works like you said. Thanks for the suggestion!

rafaelfranca commented on .gitignore:
```diff
@@ -17,3 +17,6 @@ pkg/
 /tmp/
 /yarn-error.log
 /test-reports/
+
+# local plugin
+.history/
```
This should not be here. Maybe in your global gitignore.

rafaelfranca commented:
Thank you

creaturenex commented:
Thank you so much @rafaelfranca!
